.travis.yml

```diff
@@ -28,6 +28,9 @@ matrix:
     env: SKIP_RELEASE=true
   - jdk: openjdk14
     env: SKIP_RELEASE=true MOCK_MAKER=mock-maker-inline
+  - jdk: openjdk11
+    script: ./gradlew spotlessCheck
```

Comment:
Does ^^ mean that on openjdk11 we only run "spotlessCheck" task?

Reply:
It means that we have 1 task that only runs spotless, and that task runs on opendjk11. We still have the other openjdk11 builds to verify that the tests pass etc... I will add a comment for clarification.